src/test/java/com/exasol/adapter/dialects/mysql/MySqlSqlDialectIT.java

```diff
@@ -424,8 +424,8 @@ void testRightJoinWithComplexCondition() throws SQLException {
                     + " b ON a.\"x\"||a.\"y\"=b.\"x\"||b.\"y\" ORDER BY a.\"x\"";
             final ResultSet expected = getExpectedResultSet(
                     List.of("x INT", "y VARCHAR(100)", "a INT", "b VARCHAR(100)"), //
-                    List.of("2, 'bbb', 2, 'bbb'", //
-                            "null, null, 3, 'ccc'"));
+                    List.of("null, null, 3, 'ccc'", //
+                            "2, 'bbb', 2, 'bbb'"));
```

Comment:
It will be nice when https://github.com/exasol/hamcrest-resultset-matcher/issues is there ;-)